src/components/filters/filter_icons_overlay/fitler_icons_overlay.xml

```diff
@@ -8,7 +8,10 @@
         position="getFilterHeaderPosition(header)"
         isActive="isFilterActive(header)"
         t-key="'icon_'+header.col + '_' + header.row"
-        onClick="() => this.toggleFilterMenu(header)"
+        onClick="(event) => {
+          this.toggleFilterMenu(header); 
+          event.stopPropagation();
+        }"
```

Comment:
```suggestion
        onClick.stop="() => this.toggleFilterMenu(header)"
```
You can use `stop` on the handler in order to automatically call `stopPropagation` on the event. See https://github.com/odoo/owl/blob/master/doc/reference/event_handling.md#modifiers

Reply:
@pro-odoo I tried to use `t-on-click.stop` but an error constantly shows up: 
`Uncaught Error: Invalid handler (expected a function, received: 'undefined')`
And jest tests also failed because of the same error. 
I searched on the codebase and found almost all usages of `t-on-click` is related to some original DOM components (like `div` or `button`). Here we use it in `FilterIcon` component which is defined by us. Does it probably cause issue? 
I will keep using `onClick` until finding out what goes wrong :)

Reply:
You are correct, the issue is that the onCLick in this file is actually a props pased to the component FilterIcon and not an event handling ;-)  I guess that pierre suggestion would rather address the `t-on-click`defined on the component itself.
```html
<div class="o-filter-icon" t-att-style="style" t-on-click="props.onClick">
```
You could add a `.stop` there :)